lib/rucio/tests/test_multi_vo.py

```diff
@@ -204,6 +210,17 @@ def test_rses_at_different_vos(self):
         assert_true(new in rse_list_new)
         assert_true(shr in rse_list_new)
 
+        attribute_value = generate_uuid()
+        add_rse_attribute(new, 'test', attribute_value, 'root', **self.new_vo)
+        rses_tst_1 = [r for r in get_rses_with_attribute_value('test', attribute_value, 'test', **self.vo)]
```

Comment:
`list(get_rses...)` is more concise than `[r for r in get_rses...]` (and arguably clearer)